### PR TITLE
review Schema created

### PR DESCRIPTION
**Pull request by ajkatom**
### What this PR does (required):
created review Schema

### CODE REVIEW

**Review by ezeilo-su — COMMENTED** [review]
Schema looks good!

**Comment by ajkatom** [comment]
> 

can i merge?

**Comment by ezeilo-su** [comment]
> > 
> 
> can i merge?

Please wait for approval before you can merge :smiley:

**Review by moffatethan — APPROVED** [review]

**Pull request merged**
